specs/pvs/corla.pvs

```diff
@@ -1001,23 +1050,29 @@ END rlas
 
 %+BEGIN cvrs
 
-%%% \section{Cast Vote Records}
+%%% \subsection{Cast Vote Records}
 
 %%% Cast vote records, also known as CVRs, are the digital
 %%% interpretations of paper ballot records by a computer. They
 %%% frequently, but not always, contain an interpretation of all voter
-%%% choicdes in all contests on a ballot. CVRs are sometimes
+%%% choices in all contests on a ballot. CVRs are sometimes
 %%% syntactically written as comma-separated values, and other times
 %%% in plain English. Some CVRs use a election-specific encoding
 %%% scheme to represent choices (e.g., a '1' means "John Doe"); others
 %%% use plain English.
 %%%
+%%% CVRs may (and in the case of CVRs exported from Colorado's Dominion system, do)
+%%% contain information about the ballot beyond the voter choices. For 
+%%% example, ballot style, precinct id and ballot type (in-person vs. mail)
+%%% may be included.
+%%%
 %%% CDOS requirements mandate that CVRs are exported and uploaded to
 %%% the RLA back-end as CSV files by county officials using their
 %%% voting systems and the RLA Tool, respectively.
 
 % @todo kiniry For a given election, there must be a 1-1
 % correspondence between its ballots and its CVRs.
+% @review kiniry between its ballots and the data-rows of its CVR(s), right? 
```

Comment:
I don't think we can actually always expect either a 1-1 correspondence between paper ballots and CVRs (given phantom ballots) or manifest entries and either of those (human error, to be handled also via phantom ballots).